languages/csharp-dotnet-2.2/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "artifacts": [
+        {"source":"/opt/algorithm/bin/Release/netcoreapp2.2/__ALGO__.dll", "destination":"/opt/algorithm/algo.dll"},
+        {"source":"/opt/algorithm/bin/Release/netcoreapp2.2/__ALGO__.runtimeconfig.json", "destination": "/opt/algorithm/algo.runtimeconfig.json"},
+        {"source":"/opt/algorithm/resources", "destination":"/opt/algorithm/resources"}
```

Comment:
the rust langpack was using this folder name because there's no standard place for resource files to be included in rust, but i bet there is one for c# already (which might need to get bundled in with the build command).  e.g. in scala the best practice is the src/main/resources folder.

Reply:
There are no dependencies on the finished system outside of the dll and runtimeconfig.json file, so we could probably just remove that. Kept it for posterity.